docs/Admin/Test-ExchangePropertyPermissions.md

```diff
@@ -0,0 +1,41 @@
+# Test-ExchangePropertyPermissions
+
+Download the latest release: [Update-Engines.ps1](https://github.com/microsoft/CSS-Exchange/releases/latest/download/Update-Engines.ps1)
+
+## Syntax
+
+```powershell
+Test-ExchangePropertyPermissions.ps1
+    [-TargetObjectDN] <string>
+    [-ComputerAccountDN] <string>
+    [[-DomainController] <string>]
```

Comment:
There is an extra `[ ]` in this line there a reason for that?

Reply:
It's an optional parameter, so that's how Get-Help shows it. I literally just copied this syntax from Get-Help.